site/docs/hosting/comparison.md

```diff
@@ -29,19 +29,20 @@ We therefore recommend you to use a different kind of hosting for more complex b
-| DigitalOcean Apps     | $5         |                                   |                                                                                                     | :question:         | :question:         | :question:         | Not tested                           |
-| Fastly Compute@Edge   |            |                                   |                                                                                                     |                    |                    |                    |                                      |
+| Name                   | Min. price | Pricing                                                               | Limits                                                                                  | Node.js | Deno                        | Web | Notes                          |
+| ---------------------- | ---------- | --------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------- | --------------------------- | --- | ------------------------------ |
+| Deta                   | Free       | No paid plans yet                                                     | No specific limits                                                                      | ✅       | ❓                           | ❓   |                                |
+| Deno Deploy            | Free       | $10/mo subscription for 5M req and 100 GB; $2/1M req, $0.3/GB network | [100K req/day, 100 GB/mo, 10 ms CPU-time limit](https://deno.com/deploy/pricing)        | ❌       | ✅                           | ❌   |                                |
+| DigitalOcean Functions | Free       | $1.85/100K GB-s                                                       | [90K GB-s/mo](https://docs.digitalocean.com/products/functions/details/pricing/)        | ✅       | ❌                           | ❓   |                                |
+| Cloudflare Workers     | Free       | $5/10M req                                                            | [100K req/day, 10 ms CPU-time limit](https://workers.cloudflare.com/)                   | ❌       | [✅](https://denoflare.dev/) | ✅   |                                |
+| Heroku                 | Free       | It's complicated                                                      | [550-1000 h/mo](https://www.heroku.com/pricing)                                         | ✅       | ❓                           | ❓   |                                |
+| Vercel                 | Free       | $20/mo subscription                                                   | [Unlimited invocations, 100 GB-h, 10 s time limit](https://vercel.com/pricing)          | ❓       | ❓                           | ❓   | Not intended for non-websites? |
```

Comment:
And this should move back after `Scaleway Containers`

Reply:
Ah yeah. Sorting is hard :D